src/libs/ReportUtils.js

```diff
@@ -3515,15 +3515,18 @@ function getMoneyRequestOptions(report, reportParticipants) {
 
     const participants = _.filter(reportParticipants, (accountID) => currentUserPersonalDetails.accountID !== accountID);
 
-    // Verify if there is any of the expensify accounts amongst the participants in which case user cannot take IOU actions on such report
-    const hasExcludedIOUAccountIDs = lodashIntersection(reportParticipants, CONST.EXPENSIFY_ACCOUNT_IDS).length > 0;
-    const hasSingleParticipantInReport = participants.length === 1;
-    const hasMultipleParticipants = participants.length > 1;
-
-    if (hasExcludedIOUAccountIDs) {
+    // We don't allow IOU actions if an Expensify account is a participant of the report, unless the policy that the report is on is owned by an Expensify account
```

Comment:
What's the background on why we do this?

Reply:
ah I see after looking at `CONST.EXPENSIFY_ACCOUNT_IDS`. But do you have an example of a room that's owned by these accounts where we do want to allow IOUs?

Reply:
Not sure if either of these is helpful but these are the policy and workspace chat we were testing with
policyID:9132779747441AFB
https://new.expensify.com/r/8155428955270004

Reply:
@grgia our own workspaces that employees use to submit expenses will be owned by Expensify accounts. So in order for us to be able to request reimbursement, we need to allow it when we are the policy owners.